restock.lic

```diff
@@ -34,6 +34,7 @@ class Restock
       num_needed = item['quantity'] - remaining
       buy_num = (num_needed / item['size'].to_f).ceil
       coin_needed += buy_num * item['price']
+      coin_needed += 2502 if UserVars.sun['night'] = true && town == "Shard"
```

Comment:
Could you add a comment explaining why this is necessary?

Reply:
When you go to Shard at night as a non citizen you need to bribe the night attendant to enter the shop. The bribe is 2501 normally. If you do not bribe the attendant you cannot enter the store. If you use the funds you fetched from the bank just for the runestones, or whatever restock items you are trying to get, you are then short 2501 for the items you tried to obtain. The runestone shop keeper has his dog attack you if you do not have enough money.

Reply:
Is this extra money per item to buy (i.e. inside the loop), or just a one-time cost (i.e. outside the loop)?

Reply:
It is a one time cost as long as you stay inside the shop and purchase everything at once. I need to watch restock closer, or check the logs, to see if all the items are purchased at once.